src/app/api/index.tsx

```diff
@@ -21,7 +21,7 @@ export const npm = axios.create({
     timeout: 5000,
 });
 export const roadmap = axios.create({
-    baseURL: 'https://us-central1-doc-it-api.cloudfunctions.net',
+    baseURL: 'https://raw.githubusercontent.com/pxblue/pxb-database/master/database/doc-it',
```

Comment:
is there a need to have the extra /database directory or could we have /doc-it at the root level in the repository?

Reply:
I was thinking this pxb-database repo could be extended beyond doc-it and potentially used for other things in the future.  That's why I put a doc-it bucket for now.  I think it makes sense to leave the doc-it folder for now.

Reply:
Yes, totally agree about keeping the doc-it folder. Just wondering if we should go from:
`pxb-database/database/doc-it` to
`pxb-database/doc-it`

Reply:
or maybe we call the folder 'deployed' or something. I just looked at the database repo, so I understand why there's an extra folder now.

Reply:
I updated the output folder to `deployed` instead of `database` and update doc-it to pull it down.  Surprisingly github api has been very responsive.  I think now that there aren't any service issues this might be what to expect.   At least for URL changes the updates have been instant.